testsuite/create_molecule_position.py

```diff
@@ -0,0 +1,84 @@
+import numpy as np 
```

Comment:
note that you have added two tests that do the same (or very similar) `testsuite/create_molecule_position.py` and `testsuite/create_molecule_position_test.py` please either remove one of them or if they do something different merge them into one